media/css/cms/flare-media.css

```
@@ -0,0 +1,87 @@
+/*
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at https://mozilla.org/MPL/2.0/.
+*/
+
+.media img,
+.media video,
+.media iframe {
+    display: block;
+    width: 100%;
+    height: auto;
```

@maribedran Re: the missing border radius, I think the important thing is consistency within the intro component. Right now, the media (images, video) don't have a border radius, but the images looked like they did because the placeholder image has the rounded borders inside the image file. Assuming that both images and videos in the intro component should have a border radius, adding it here should fix both.

```suggestion
    height: auto;
    border-radius: calc(var(--scale-16) * 1px);
```